charts/tfy-karpenter-config/templates/karpenter-gpu-default-awsnodetemplate.yaml

```diff
@@ -15,6 +15,7 @@ spec:
   {{- range $key, $val := .Values.karpenter.gpuDefaultNodeTemplate.extraTags }}
     {{ $key }}: {{ $val }} 
   {{- end }}
+  amiFamily: {{ .Values.karpenter.gpuDefaultNodeTemplate.amiFamily }}
   blockDeviceMappings:
     - deviceName: /dev/xvda
```

Comment:
This needs an if/else 
AL2 = xvda
Bottlerocket = xvdb

I am not entirely sure if we should do it for other node templates too